snmp/datadog_checks/snmp/data/default_profiles/audiocodes-mediant-sbc.yaml

```diff
@@ -0,0 +1,7 @@
+extends:
+  - _base.yaml
+  - _generic-if.yaml
+  - _generic-host-cpu-memory.yaml
```

Comment:
wondering if it's a good idea to remove the metrics definition in the original profile and just replace with this since it's not exactly the same